system-int-test/src/cmr/system_int_test/utils/generic_util.clj

```diff
@@ -45,7 +45,8 @@
         :connection-manager (sys/conn-mgr)
         :body (when document (json/generate-string document))
         :throw-exceptions false
-        :headers {transmit-config/token-header token}}
+        :headers {"Accept" "application/json"
```

Comment:
your version of this function is not exactly the same as mine. I can move over to yours if they were the same.

Reply:
I don't understand, these changes here ARE moving yours over to mine? as in, i made the needed changes in this PR here. do you mean you are doing more work independently and would like to open a PR ahead of this merge? will contact you for elaboration.

Reply:
We talked and realized that funny enough, I missed a new update that occurred while I was in the middle of rebasing this branch onto the updates that i had. 

Will rebase again on to new changes and consolidate this namespace/requester use.